vnext/Microsoft.ReactNative/ReactNativeHost.cpp

```diff
@@ -52,6 +52,24 @@ IAsyncAction ReactNativeHost::LoadInstance() noexcept {
   return ReloadInstance();
 }
 
+::Microsoft::ReactNative::ReactPropertyId<::Microsoft::ReactNative::ReactNonAbiValue<winrt::weak_ref<ReactNativeHost>>>
+ReactNativeHostProperty() noexcept {
+  static ::Microsoft::ReactNative::ReactPropertyId<
+      ::Microsoft::ReactNative::ReactNonAbiValue<winrt::weak_ref<ReactNativeHost>>>
+      propId{L"ReactNative.ReactNativeHost", L"ReactNativeHost"};
+  return propId;
+}
+
+/*static*/ winrt::Microsoft::ReactNative::ReactNativeHost ReactNativeHost::GetReactNativeHost(
+    ReactPropertyBag const &properties) noexcept {
+  if (auto wkHost = properties.Get(ReactNativeHostProperty()).Value()) {
```

Comment:
>.Value() [](start = 61, length = 8)

We should account for a missing property value.
The simplest way is to add another "if (auto ..."